### DISCUSSION TITLE
打包之后图标问题

### DISCUSSION THREAD

**Discussion by airyland** [tw93/Pake#413] [General]
命令是：`pake https://ai.ls --name AILS --transparent --multi-arch --user-agent ai-ls-desktop-0.01 --icon 'ICNS PATH' --show-menu`

打包之后自动打开的安装窗口，bar 图标正常，但是里面的是默认图标。

<img width="995" alt="image" src="https://user-images.githubusercontent.com/559179/235718775-628baa8c-70d5-4929-9fdf-005508237c8d.png">

**Comment by airyland** [comment]
使用 Action 打包没有这个问题，先关掉这个讨论。

**Reply by tw93** [reply]
谢谢，后面我们排查一下你说的这个问题
